src/Components/Carousel.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useState } from 'react'
+
+interface CarouselProps {
+  children: React.ReactNode
+}
+
+const CarouselContainer = styled.div`
+  display: grid;
+`
+
+const Selector = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 1rem;
+`
+
+const Movement = styled.button`
+  background: none;
+  border: none;
+  color: black;
```

Comment:
Extra color here - I tend to alphabetise the rules and then move order up/down for overrides in longer CSS blocks